apps/src/code-studio/pd/application_dashboard/application_dashboard.jsx

```diff
@@ -0,0 +1,61 @@
+/* global window */
+
+/**
+ * Main landing page and router for the application dashboard.
+ */
+import React from 'react';
+import Header from '../components/header';
+import {
+  Router,
+  Route,
+  IndexRedirect,
+  useRouterHistory
+} from 'react-router';
+import {createHistory} from 'history';
+import Summary from './summary';
+import QuickView from './quick_view';
+
+const ROOT_PATH = '/pd/application_dashboard';
+const browserHistory = useRouterHistory(createHistory)({
+  basename: ROOT_PATH
+});
+
+const ApplicationDashboard = ()=> (
+  <Router history={browserHistory} >
+    <Route path="/" breadcrumbs="Application Dashboard" component={Header}>
+      <IndexRedirect to="/summary"/>
+      <Route
+        path="summary"
+        breadcrumbs="Application Dashboard,Summary"
+        component={Summary}
+      />
+      <Route
+        path="csd_facilitators"
+        breadcrumbs="Application Dashboard,CSD Facilitator Applications"
+        component={QuickView}
+        title="CSD Facilitator Applications"
+      />
+      <Route
+        path="csp_facilitators"
+        breadcrumbs="Application Dashboard,CSP Facilitator Applications"
+        component={QuickView}
+        title="CSP Facilitator Applications"
+      />
+      <Route
+        path="csd_teachers"
+        breadcrumbs="Application Dashboard,CSD Teacher Applications"
+        component={QuickView}
+        title="CSD Teacher Applications"
+      />
+      <Route
+        path="csp_teachers"
+        breadcrumbs="Application Dashboard,CSP Teacher Applications"
+        component={QuickView}
+        title="CSP Teacher Applications"
+      />
+    </Route>
+  </Router>
+);
+
+export default ApplicationDashboard;
+window.ApplicationDashboard = ApplicationDashboard;
```

Comment:
This and the `/* global window */` on line 1 are not needed. Please remove. 

This was leftover in the workshop dashboard from before we had organized entry points. This `window.ApplicationDashboard` is never used, and instead the react component is imported and rendered (correctly) in the entry point [here](https://github.com/code-dot-org/code-dot-org/pull/18265/files#diff-5e91ab1ea5bf160ce14faa5dfdbec338R7).